roles/rundeck/tasks/rundeck-cli.yaml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Download the Rundeck CLI inside the container
+  ansible.builtin.shell: podman exec '{{ rundeck_container_name }}' wget {{ rundeck_cli_url }}
```

Comment:
I was expecting it to be mounted as a path

Reply:
I thought about that option too. How about getting the DEB inside the container then install it?

Reply:
Why? Just mount the path else you are forcing one of the following:
- maintaining a custom image
- installing the deb every time the container starts and allowing root access

Reply:
I didn't go with DEB. Mounted the JAR instead then alias rd to `podman run --rm --it` it